examples/with-eip-1193-provider/.env.local.example

```diff
@@ -0,0 +1,6 @@
+NEXT_PUBLIC_RPC_URL="<The public rpc url i.e. infura, alchemy>"
```

Comment:
I think an example here would be nice to show the format we expect. When running locally I set this to be `https://sepolia.infura.io/v3/84842078b09946638c03157f83405213` (community infura key, sepolia). Would this work as a default?